views/containers/Game.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import keydrown from 'keydrown';
+import GameScene from '../components/GameScene';
+import Map from '../components/Map';
+import Tree from '../components/Tree';
+import Character from '../containers/Character';
+import Exit from '../components/Exit';
+import { characterMoveToPoint } from '../actions/Character';
+
+const ReactTHREE = require('react-three');
+
+const Object3D = ReactTHREE.Object3D;
+
+class GameComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.loop = null;
+    this.moveTarget = props.characterPosition;
+  }
+
+  componentDidMount() {
+    // TODO create a component that handles the loop, game should dispatch an action for it to start/stop looping
+    this.loop = setInterval(() => {
```

Comment:
You might want to check this library as well: https://github.com/IceCreamYou/MainLoop.js